### PR TITLE
Add option to specify bat target in run-benchmarks.sh

### PR DESCRIPTION
**Pull request by eth-p**
The benchmarking script gets kind of tedious when you have to keep changing the PATH variable. I updated it to accept `--debug`, `--release`, `--system`, and `--bat=path` as a way to select which target to perform benchmarks on.

### CODE REVIEW

**Comment by sharkdp** [comment]
This looks great, but I hope that we would never run the benchmarks on the debug version, right? Can we remove the `--debug` part, or do you think that could be useful for something?

**Review by sharkdp — COMMENTED** [review]
This looks great, but I hope that we would never run the benchmarks on the debug version, right? Can we remove the `--debug` part, or do you think that could be useful for something?

File: tests/benchmarks/run-benchmarks.sh [hunk 1]
```diff
@@ -8,21 +7,47 @@ if ! which hyperfine > /dev/null 2>&1; then
...
+	echo "You can use '--system', '--debug', or '--release'."
+	exit 1
+fi
+
+if ! command -v "$BAT" &>/dev/null; then
+	echo "Could not find the build of bat to benchmark."
+	case "$BAT" in
+		"bat")                        echo "Make you sure to symlink 'batcat' as 'bat'." ;;
+		"../../target/debug/debug")   echo "Make you sure to 'cargo build' first." ;;
+		"../../target/debug/release") echo "Make you sure to 'cargo build --release' first." ;;
```
Comment (sharkdp) [inline tests/benchmarks/run-benchmarks.sh:31]: those paths seem wrong, right? There is one additional `/debug/`

Comment (sharkdp) [reply tests/benchmarks/run-benchmarks.sh:31]: By the way, for me, those paths would not work. I have a custom
```toml
[build]
target-dir = "/home/shark/.cargo-target"
```
in `~/.cargo/config` (which I can only recommend).

**Comment by sharkdp** [comment]
Closing this for now, but happy to reopen if someone wants to make progress on this.

**Comment by eth-p** [comment]
I seem to have missed all the comments on this... whoops. I'll see if I can work on it right now.

File: tests/benchmarks/run-benchmarks.sh (hunk 1 above)
Comment (eth-p) [reply tests/benchmarks/run-benchmarks.sh:31]: Fixed in 3ed0081f1fb4b707af026fcf1789909114487975.

**Review by sharkdp — APPROVED** [review]
Thanks!

**Comment by sharkdp** [comment]
There is a small merge conflict, but apart from that this looks good.

**Comment by eth-p** [comment]
Sorry for the delay! The conflict should be fixed now.

**Pull request merged**